dataflux_pytorch/benchmark/lightning_checkpoint_benchmark.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import torch
+
+from typing import Tuple
+from torch.utils.data import DataLoader
+from torch import Tensor
+from torch.utils.data import DataLoader
+from lightning import Trainer
+from lightning.pytorch.callbacks import ModelCheckpoint
+from lightning.pytorch.demos import WikiText2, Transformer
+from lightning.pytorch import LightningModule
+from lightning.pytorch.plugins.io import TorchCheckpointIO
+
+from dataflux_pytorch.lightning import DatafluxLightningCheckpoint
+
+class LightningTransformer(LightningModule):
+    def __init__(self, vocab_size: int = 33278, nlayers: int = 100) -> None:
+        super().__init__()
+        self.model = Transformer(vocab_size=vocab_size, nlayers=nlayers)
+
+    def forward(self, inputs: Tensor, target: Tensor) -> Tensor:
+        return self.model(inputs, target)
+
+    def training_step(self, batch: Tuple[Tensor, Tensor], batch_idx: int) -> Tensor:
+        inputs, target = batch
+        output = self(inputs, target)
+        loss = torch.nn.functional.nll_loss(output, target.view(-1))
+        return loss
+
+    def configure_optimizers(self) -> torch.optim.Optimizer:
+        return torch.optim.SGD(self.model.parameters(), lr=0.1)
+
+    def prepare_data(self) -> None:
+        WikiText2(download=True)
+
+    def train_dataloader(self) -> DataLoader:
+        dataset = WikiText2()
+        return DataLoader(dataset)
+
+
+def main(project: str, bucket: str, ckpt_dir_path: str, save_only_latest: bool, dataflux_ckpt: bool, model_size: int = 100, steps: int = 5):
```

Comment:
instead of model_size, would it make sense to call the parameter nlayers or something that more directly refers to what it's setting?